### PR TITLE
robins suggestions

### PR DESCRIPTION
**Pull request by RobertAKARobin**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -30,48 +32,40 @@ Your app must:
...
 git checkout -b gh-pages
 ```
 #### 2. Initial deploy to origin (via push):
 ```
-git push --set-upstream  # defaults to "origin gh-pages"
+git push -u origin gh-pages
 ```
 #### 3. Verify
-Browse to `<your_github_name>.github.io/<repo_name>`.  Don't forget to add this to your repo url and readme.
+Browse to `<your_github_username>.github.io/<repo_name>`
```
Comment (mattscilipoti) [inline readme.md:61]: Students regularly forgot to add it as their repo url and to their readme.  Why remove this comment?

File: readme.md [hunk 2]
```diff
@@ -30,48 +32,40 @@ Your app must:
...
-
-tldr;
+**DO NOT** follow the instructions provided by GitHub for creating your Project Pages. Why not? The instructions expect us to host our application on a custom web domain (yourname.com).
 
-1. Create a branch from "master" named "gh-pages".
-2. Push "gh-pages" to "origin".
-3. Browse to `<your_github_name>.github.io/<repo_name>`.
+Instead, try these steps:
 
-Why? They expect GitHub Project pages to provide an, easy way to create a separate information site for our application.  They expect us to host our application somewhere else (with robust servers, db, custom domain, etc.).  We just want our project to be available here.
```
Comment (mattscilipoti) [inline readme.md:51]: Please explain why they do not follow the "normal" github pages instructions.  Otherwise, it's just magic.

**Comment by adambray** [comment]
:+1:
